fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1377,6 +1387,22 @@ public void setEnablePruneIcebergManifest(boolean enablePruneIcebergManifest) {
         this.enablePruneIcebergManifest = enablePruneIcebergManifest;
     }
 
+    public boolean enableReadIcebergPuffinNdv() {
+        return enableReadIcebergPuffinNdv;
+    }
+
+    public void setEnableReadIcebergPuffinNdv(boolean enableReadIcebergPuffinNdv) {
+        this.enableReadIcebergPuffinNdv = enableReadIcebergPuffinNdv;
+    }
+
+    public boolean enableIcebergColumnStatistics() {
+        return enableIcebergColumnStatistics;
+    }
+
+    public void setEnableIcebergColumnStatistics(boolean enableIcebergColumnStatistics) {
+        this.enableIcebergColumnStatistics = enableIcebergColumnStatistics;
+    }
+
     public boolean isCboPredicateSubfieldPath() {
         return cboPredicateSubfieldPath;
     }
```

Comment:
Consistent method naming: Your getter methods should follow the general Java getter pattern. Instead of enableReadIcebergPuffinNdv(), prefer using isEnableReadIcebergPuffinNdv() or just getEnableReadIcebergPuffinNdv(). The same applies for other similar methods.